### PR TITLE
Create contribution2.json

### PR DESCRIPTION
**Pull request by nicolaimg**

### CODE REVIEW

**Comment by shiffman** [comment]
Hi! Thank you for this submission! Were you at PCD CPH this evening?

File: content/videos/challenges/19-superellipse/showcase/contribution2.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Piet Hein Rainbow",
+  "url": "https://editor.p5js.org/nicolaimg/full/q-kLyOogz",
+  "author": {
+    "name": "Nicolai Munch Grundsøe",
+    "url": "ig: @nicolai.gif"
```
Comment (shiffman) [inline content/videos/challenges/19-superellipse/showcase/contribution2.json:6]:
```suggestion
    "url": "https://www.instagram.com/nicolai.gif/"
```

This has to be a URL unfortunately for it to work!

**Comment by shiffman** [comment]
Would you mind trying to upload an image as well? You can name it `contribution2.png`!

image

If it's confusing how to do so just let me know, I'll help!!

**Comment by nicolaimg** [comment]
Yes, i were! Thanks for the great workshop!

**Comment by nicolaimg** [comment]
I don't know how to add the png to the pull request.
First time using GitHub...

**Comment by shiffman** [comment]
You did it!!! ❤️ I'll hopefully merge this tomorrow, there are a lot of builds in the queue!

**Pull request merged**